.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+
+### Prettier
+- It is a good idea to use format-on-save
+
+## Creating pull-requests 
+<!-- This section is responsible for guiding contributors on how they should submit pull-requests in a way that ensures the most swift process and teach them how to catch low-hanging issues there might come up in code-review. -->
+
+### Before you submit  
+- Make sure you have completed the steps in: [Setup tooling for contribution](#setup-tooling-for-contribution)
+- Make sure you have written tests according to the guidelines described in [The good: test](#test)
+- Do a self-review <!-- We could link to self-review in the templates and have people check it off? -->
+
+### After you submit 
+- Post in Kirby-guild 
+- Code reviews are meant to align your code with existing code 
+  - While your implementation might be perfectly valid, we are aiming for uniformity across the codebase. 
+- Don't commit and exit - make sure to stay responsive and come back to answer comments on your code review
+
+## The good:
+<!-- The following sections have the responsibility of explaining contributors our best pratices such that they can more easily have their changes implemented in kirby --> 
+<!-- Ideally we should be able to refer to these sections if people break the guidelines in their issue or pull-requests, to make sure we are all preaching the same guidelines -->
+
+### Issue 
+- General properties we are looking for and how to achieve them 
+  - Readability 
+  - Easily understandable 
+    - No internal lingo 
+  - Proof-read
+  - Self-contained
+    - Screenshots if necessary
+    <!-- I think the following point is important if Kirby is to be spread out amongst several teams -->
+    - Don't assume people have knowledge outside of Kirby - contributors might come from many different projects 
+
+### Self-review 
+- Don't change files that is not related to your pull-request
+  - for example don't reorder imports in files that is not directly related to the issue at hand
+- Check for unused imports
+- Other self review points go here
+
+### Pull-request
+- How is a pull-request that is easily reviewed constructed?
+
+### Test
+- How should testing be done 
+
+## I have a question related to Kirby
+<!-- This section is responsible for teaching contributors how to get help with Kirby, we should probably link to this section from the readme.md -->
+We have **Kirby Open Office** every Tuesday and Thursday between 9:15 to 10:15. During these timeslots we will answer questions asked in the @kirby-design/kirby-guild slack channel and inquiry requests submitted on github. You are always welcome to ask questions outside these timeslots and we will answer it when we got time. 
+
+**Do not send direct messages to members of team-kirby:** This is not because we do not want to talk to you - we however prefer questions to be asked in public, such that everyone can benefit from the answers. 
```

Comment:
```Do not send direct messages to members of team-kirby:** This is not because we do not want to talk to you - we do however prefer questions to be asked in public, so that everyone can benefit from the answers.
```